resources/js/forms/PlotForm.vue

```diff
@@ -341,6 +341,12 @@
             })
             console.error(e)
           }
+          for (const value in this.form.errors.errors) {
+                  this.$notify({
+                      type: 'error',
+                      text: this.form.errors.errors[value][0],
+              })
+          }
```

Comment:
the errors object has a toArray method that would make this much simpler. Also I think using $alert here might be more appropriate since it forces the user to pay attention.

```suggestion
          this.$alert(this.form.errors.toArray().join(' '))
```